client/me/controller.js

```diff
@@ -303,7 +302,7 @@ export default {
 
 		if ( isWelcome ) {
 			ReactDom.unmountComponentAtNode( document.getElementById( 'secondary' ) );
-			context.layout.setState( { noSidebar: true } );
+			context.store.dispatch( setSection( null, { hasSidebar: false } ) );
```

Comment:
This makes me wonder whether we should have separate actions for `sidebar` and `isLoading`, since they're not always tied to the section as is the case here.